services-js/commissions-search/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "commissions-search-ui",
+  "version": "0.1.0",
+  "private": true,
+  "engines": {
```

Comment:
You can clear out the engines, which will make the tests not error out so early.